notes/randomNote/views.py

```diff
@@ -26,6 +26,7 @@ def get(self, request, id):
         for image in images:
             with open(settings.ATTACHMENTS_DIR + "/" + image.file_path, 'rb') as f:
                 image_base64 = base64.b64encode(f.read()).decode('utf-8')
-            image_data.append(image_base64)
+                full_image_base64 = f"data:image/png;base64,{image_base64}"
```

Comment:
we moved this logic from frontend to backend

i propose frontend to accept image source and not to know what kind of source it actually is (base64, url etc). We would be able to change image source on backend side with no changes for the frontend